src/components/layouts/index.js

```diff
@@ -133,12 +133,9 @@ const LayoutWrapper = ({ children }) => {
   }
 
   return (
-    // eslint-disable-next-line node/handle-callback-err
-    <Sentry.ErrorBoundary
-      fallback={({ error }) => <ErrorFallback error={error} />}
-    >
+    <ErrorBoundary>
       <Layout>{children}</Layout>
-    </Sentry.ErrorBoundary>
+    </ErrorBoundary>
```

Comment:
@simon-debruijn did you intentionally not wrap the `/login`, `/404`, and `/500` pages?

Reply:
The **404** and **500** pages don't seem that logical to wrap with a ErrorBoundary necessarily, since they both appear when an error occurs. The login page seems like a smart thing to wrap with the ErrorBoundary indeed.

If we als add a ErrorBoundary around login, make sure to test if this doesn't cause any issues though 🙂 The ErrorFallback layout will be shown without sidebar if I'm correct.

Reply:
I gonna first get this one merged. We can test that behaviour in another PR